typing/typecore.ml

```diff
@@ -1336,15 +1458,17 @@ and type_pat_aux ~exception_allowed ~constrs ~labels ~no_existentials ~mode
         pat_attributes = sp.ppat_attributes;
         pat_env = !env }
       in
-      if constrs = None then
+      begin match mode with
+      | Normal ->
         k (wrap_disambiguate "This record pattern is expected to have"
```

Comment:
I believe the house style is to indent match-branches by 4, not 2.

Reply:
I'm so glad we have someone able to notice this.